options.go

```diff
@@ -91,6 +91,7 @@ func RequestHeader(header string) Option {
 func FieldName(name string) Option {
 	return func(cs *csrf) error {
 		cs.opts.FieldName = name
+		fieldName = name
```

Comment:
Thank you for this very well documented answer.

I use a similar helper to inject the html fragment in my templates and it works very well, but I was confused by the option documentation: https://github.com/goji/csrf/blob/master/options.go#L89L90. 

Adding `cs.c.Env[fieldName] = cs.opts.FieldName` doesn't work because it override `cs.c.Env[tokenKey] = mask(realToken, cs.c, r)` as `fieldName = tokenKey`. That's why I added an additional key in my first commit.

Maybe it's just a documentation issue and I don't want to make you loose more time on that detail.
Let me know.